parcel.go

```diff
@@ -24,9 +26,28 @@ type ParcelParams struct {
 	PhoneNumber      string
 	ExternalID       string
 	ToServicePointID int64
-	Weight           int64
 	OrderNumber      string
 	SenderID         int64
+
+	// Weight contains the weight in Kilograms. Supports up to 3 decimals
+	// (grams). For example: decimal.New(20, -3) specifies 20 grams.
+	Weight decimal.Decimal
+
+	CustomsInvoiceNr        string
+	CustomsShipmentType     int64
```

Comment:
Maybe this package should define the possible values as a typed const (enum-esque construction)?